core/server/src/main/java/alluxio/master/block/BlockMasterClientRestServiceHandler.java

```diff
@@ -45,6 +45,9 @@
 
   private final BlockMaster mBlockMaster = AlluxioMaster.get().getBlockMaster();
 
+  // prevent instantiation
+  private BlockMasterClientRestServiceHandler() {}
```

Comment:
this needs to have a public constructor because it is instantiated by jersey framework when creating REST endpoint handlers